<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1656345273234' stop='1656345301019'>
    <name>Test Verify Industry Contact</name>
    <title>Test Verify Industry Contact</title>
    <test-cases>
        <test-case start='1656345273276' status='failed' stop='1656345300897'>
            <name>Verify that user is able to add industries </name>
            <title>Verify that user is able to add industries </title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Timed out retrying after 4000ms: Expected to find element: `:nth-child(1) > .card > .card-body > :nth-child(6) > .badge`, but never found it.</message>
                <stack-trace>AssertionError: Timed out retrying after 4000ms: Expected to find element: `:nth-child(1) > .card > .card-body > :nth-child(6) > .badge`, but never found it.
                    at Contacts.verifyUserIsAbleToAddIndustries (https://design-portal-oscar-cannect-dev.oscarintelligence.net/__cypress/tests?p=cypress\integration\ContactsAndIndustryContacts\TestVerifyUserIsAbleToAddIndustries.js:348:8)
                    at Context.eval (https://design-portal-oscar-cannect-dev.oscarintelligence.net/__cypress/tests?p=cypress\integration\ContactsAndIndustryContacts\TestVerifyUserIsAbleToAddIndustries.js:1503:22)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>